recipe/meta.yaml

```diff
@@ -11,25 +11,41 @@ source:
   url: http://glaros.dtc.umn.edu/gkhome/fetch/sw/metis/metis-5.1.0.tar.gz
 
 build:
-  skip: true  # [win]
   number: 3
   binary_relocation: true
+  features:
+    - vc9  # [win and py27]
+    - vc10  # [win and py34]
+    - vc14  # [win and py35]
 
 requirements:
   build:
     - cmake
     - toolchain
+    - python  # [win]
```

Comment:
Adding to the `build` and `run` section is probably good practice. One day we may drop the `python` dependency and use only the `vc`s. (But for now you still need both or the re-rendering will mess up the build matrix.)

```yaml
   - vc 9  # [win and py27]
    - vc 10  # [win and py34]
    - vc 14  # [win and py35]
```